### DISCUSSION TITLE
How agent sync/update is working ?

### DISCUSSION THREAD

**Discussion by BornTKill** [schorschii/oco-agent#21] [Q&A] [answered]
Hello @schorschii 

I noticed (at least on osx) when the agent is started as a deamon that update/sync is not always up to date. Have to unload and load daemon (launchctl command) to force sync/update. Don’t know how to debug this behaviour.

Regards.

**Comment by schorschii** [comment]
Update of the device info ("inventory") is not done on every agent connection because it would cause too much CPU load to query logins, installed software etc. every minute. You can adjust the agent update interval in the server settings. Please have a look at the [docs](https://github.com/schorschii/OCO-Server/blob/master/docs/Computers.md#updating-computer-inventory-values).

Please use the Github Discussions for such questions.

**Reply by schorschii** [reply]
Does this solved your problem, @BornTKill ?

**Reply by schorschii** [reply]
Does this solved your problem, @BornTKill ?

**Comment by BornTKill** [comment, marked as answer]
Thank you @schorschii
